packages/frontend/core/src/blocksuite/presets/ai/ai-spec.ts

```diff
@@ -17,140 +17,164 @@ import {
   ParagraphBlockSpec,
 } from '@blocksuite/blocks';
 import { assertInstanceOf } from '@blocksuite/global/utils';
-import { literal, unsafeStatic } from 'lit/static-html.js';
+import { literal, type StaticValue, unsafeStatic } from 'lit/static-html.js';
 
 import { buildAIPanelConfig } from './ai-panel';
 import { setupCodeToolbarEntry } from './entries/code-toolbar/setup-code-toolbar';
 import {
   setupEdgelessCopilot,
-  setupEdgelessElementToolbarEntry,
+  setupEdgelessElementToolbarAIEntry,
 } from './entries/edgeless/index';
-import { setupFormatBarEntry } from './entries/format-bar/setup-format-bar';
-import { setupImageToolbarEntry } from './entries/image-toolbar/setup-image-toolbar';
-import { setupSlashMenuEntry } from './entries/slash-menu/setup-slash-menu';
+import { setupFormatBarAIEntry } from './entries/format-bar/setup-format-bar';
+import { setupImageToolbarAIEntry } from './entries/image-toolbar/setup-image-toolbar';
+import { setupSlashMenuAIEntry } from './entries/slash-menu/setup-slash-menu';
 import { setupSpaceEntry } from './entries/space/setup-space';
 
-export const AIPageRootBlockSpec: BlockSpec = {
-  ...PageRootBlockSpec,
-  view: {
-    ...PageRootBlockSpec.view,
-    widgets: {
-      ...PageRootBlockSpec.view.widgets,
-      [AFFINE_AI_PANEL_WIDGET]: literal`${unsafeStatic(
-        AFFINE_AI_PANEL_WIDGET
-      )}`,
+const pageWidgets: Record<string, StaticValue> = {
+  ...PageRootBlockSpec.view.widgets,
+};
+
+export const createAIPageRootBlockSpec = (enableAI?: boolean): BlockSpec => {
+  if (enableAI) {
```

Comment:
can we just not calling this function when enableAI is false？

Reply:
@L-Sun will follow up on these issues.

Reply:
Done